ent migrations fail

We got the following blocking error in the CI https://github.com/chainloop-dev/chainloop/actions/runs/10143109534/job/28043796679?pr=1141

```
Run wget -q https://release.ariga.io/atlas/atlas-linux-amd64-latest -O /tmp/atlas
make: Entering directory '/home/runner/work/chainloop/chainloop/app/controlplane'
atlas migrate hash --dir file://pkg/data/ent/migrate/migrations
atlas migrate diff --dir file://pkg/data/ent/migrate/migrations --to "ent://pkg/data/ent/schema" --dev-url "docker://postgres/15/test?search_path=public"
Error: loading schema: entc/load: # command-line-arguments
Error: .entc/entc_github.com_chainloop-dev_chainloop_app_controlplane_pkg_data_ent_schema_1[7](https://github.com/chainloop-dev/chainloop/actions/runs/10143109534/job/28043796679?pr=1141#step:5:8)22251277.go:167:24: fd.Deprecated undefined (type *"entgo.io/ent/schema/field".Descriptor has no field or method Deprecated)
Error: .entc/entc_github.com_chainloop-dev_chainloop_app_controlplane_pkg_data_ent_schema_1722251277.go:16[8](https://github.com/chainloop-dev/chainloop/actions/runs/10143109534/job/28043796679?pr=1141#step:5:9):24: fd.DeprecatedReason undefined (type *"entgo.io/ent/schema/field".Descriptor has no field or method DeprecatedReason)
Error: .entc/entc_github.com_chainloop-dev_chainloop_app_controlplane_pkg_data_ent_schema_1722251277.go:207:26: undefined: ent.Viewer
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -41,6 +41,8 @@ jobs:
       # This could happen if the developer ran make generate but didn't run make migration_sync
       - name: Generate migrations
         if: ${{ matrix.app == 'controlplane' }}
+        env:
+          ATLAS_VERSION: v0.24.1
         run: |
           wget -q https://release.ariga.io/atlas/atlas-linux-amd64-latest -O /tmp/atlas
           sudo install /tmp/atlas /usr/local/bin/atlas

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -44,7 +44,7 @@ jobs:
         env:
           ATLAS_VERSION: v0.24.1
         run: |
-          wget -q https://release.ariga.io/atlas/atlas-linux-amd64-latest -O /tmp/atlas
+          wget -q https://release.ariga.io/atlas/atlas-linux-amd64-$ATLAS_VERSION -O /tmp/atlas
           sudo install /tmp/atlas /usr/local/bin/atlas
 
           make -C app/controlplane migration_sync

